src/connectors/piped.ts

```diff
@@ -10,11 +10,15 @@ Connector.getArtistTrack = () => {
 	return Util.processYtVideoTitle(text);
 };
 
-Connector.getCurrentTime = () =>
-	(document.querySelector('.shaka-video') as HTMLVideoElement).currentTime;
-
-Connector.getDuration = () =>
-	(document.querySelector('.shaka-video') as HTMLVideoElement).duration;
+Connector.getCurrentTime = () => {
+	const videoEl = (document.querySelector('.shaka-video') as HTMLVideoElement);
+	return videoEl ? videoEl.currentTime : null;
```

Comment:
Would using the `?.` operator be a more concice way of expressing this work?

```
Connector.getCurrentTime = () => (document.querySelector('.shaka-video') as HTMLVideoElement)?.currentTime;
```

Reply:
It should be, yes. I wasn't sure whether the `?.` operator was discouraged by the project guidelines so I opted for the more verbose version. The only difference is that, with the `?.` operator, `undefined` will be returned if `querySelector` doesn't find the element, while the verbose version explicitly returns `null` in such case, and I don't know if that makes a difference to the core. If it doesn't, they should be functionally equivalent

Reply:
Looks like there is one linting issue to fix.

Reply:
I'd really like to fix it but I have no clue on how to do that...

Reply:
If you run `npx prettier . --write` it should fix the issues automatically